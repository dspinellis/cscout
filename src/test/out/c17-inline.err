Processing workspace TestWS
Processing directory test/c
Entering directory test/c
Processing project Prj1
Processing file c17-inline.c
Done processing file c17-inline.c
Done processing project Prj1
Processing project Prj2
Processing file c17-inline.c
Done processing file c17-inline.c
Processing file prj2.c
Done processing file prj2.c
Done processing project Prj2
Exiting directory test/c
Done processing directory test/c
Done processing workspace TestWS
Processing identifiers
2%5%8%10%13%16%18%21%24%27%29%32%35%37%40%43%45%48%51%54%56%59%62%64%67%70%72%75%78%81%83%86%89%91%94%97%100%
