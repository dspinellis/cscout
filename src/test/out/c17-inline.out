Table: Ids
EID	NAME	READONLY	UNDEFMACRO	MACRO	FUNMACRO	MACROARG	CPPCONST	CPPSTRVAL	DEFCCONSTVAL	NOTDEFCCONSTVAL	EXPCCONSTVAL	NOTEXPCCONSTVAL	ORDINARY	SUETAG	SUMEMBER	LABEL	TYPEDEF	ENUM	YACC	FUN	CSCOPE	LSCOPE	UNUSED
79	foo	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	1	0
91	MAXDIGIT	0	0	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	1
179	filename	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
321	x	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
399	modes	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
421	main	1	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	1	0
541	qqq	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
649	filename	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
869	modes	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
1119	foo2	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	1	0	0
1229	filename	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
1449	modes	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
1779	filename	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
1999	modes	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
2219	foo3	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	1	0
2329	filename	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
2549	modes	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
2849	filename	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
2975	__DATE__	1	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	1
3069	modes	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
3289	foo4	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	1	0
3295	__TIME__	1	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	1
3399	filename	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
3575	__FILE__	1	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	1
3619	modes	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
3865	__LINE__	1	0	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	1
3919	filename	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
4055	__STDC__	1	0	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	1
4139	modes	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
4389	foo5	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	1	0	0
4499	filename	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
4719	modes	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
5049	filename	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
5269	modes	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
5355	_cscout_dummy1	1	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	1	0	0
Table: Tokens
FID	FOFFSET	EID
2	297	2975
2	329	3295
2	357	3575
2	386	3865
2	405	4055
2	471	421
2	535	5355
2	558	5355
4	7	79
4	17	179
4	39	399
4	54	79
4	64	649
4	86	869
4	111	1119
4	122	1229
4	144	1449
4	166	1119
4	177	1779
4	199	1999
4	221	2219
4	232	2329
4	254	2549
4	273	2219
4	284	2849
4	306	3069
4	328	3289
4	339	3399
4	361	3619
4	380	3289
4	391	3919
4	413	4139
4	438	4389
4	449	4499
4	471	4719
4	493	4389
4	504	5049
4	526	5269
5	8	91
5	31	321
5	41	421
5	53	541
5	81	79
Table: Rest
FID	FOFFSET	CODE
2	287	\u0000a\u0000a#define 
2	305	  
2	320	\u0000a#define 
2	337	 
2	348	\u0000a#define 
2	365	 
2	377	\u0000a#define 
2	394	 1\u0000a#define 
2	413	 1\u0000a\u0000a
2	466	\u0000aint 
2	475	();\u0000a
2	522	\u0000astatic void 
2	549	(void) { 
2	572	(); }\u0000a
3	152	\u0000a\u0000a\u0000a#pragma includepath 
3	195	\u0000a
3	239	\u0000astatic void _cscout_dummy2(void) { _cscout_dummy2(); }\u0000a
4	0	inline 
4	10	(char *
4	25	, const char *
4	44	);\u0000ainline 
4	57	(char *
4	72	, const char *
4	91	) { }\u0000astatic inline 
4	115	(char *
4	130	, const char *
4	149	);\u0000astatic inline 
4	170	(char *
4	185	, const char *
4	204	) { }\u0000ainline int 
4	225	(char *
4	240	, const char *
4	259	);\u0000ainline int 
4	277	(char *
4	292	, const char *
4	311	) { }\u0000aint inline 
4	332	(char *
4	347	, const char *
4	366	);\u0000aint inline 
4	384	(char *
4	399	, const char *
4	418	) { }\u0000ainline static 
4	442	(char *
4	457	, const char *
4	476	);\u0000ainline static 
4	497	(char *
4	512	, const char *
4	531	) { }\u0000a
5	0	#define 
5	16	 11\u0000aextern int 
5	32	;\u0000aextern 
5	45	();\u0000aint 
5	56	;\u0000a\u0000a
5	80	\u0000a
5	84	() {\u0000a	
5	107	}\u0000a\u0000a
Table: Projects
PID	NAME
23	unspecified
24	Prj1
25	Prj2
Table: IdProj
EID	PID
79	24
179	24
399	24
421	24
649	24
869	24
1119	24
1229	24
1449	24
1779	24
1999	24
2219	24
2329	24
2549	24
2849	24
2975	24
3069	24
3289	24
3295	24
3399	24
3575	24
3619	24
3865	24
3919	24
4055	24
4139	24
4389	24
4499	24
4719	24
5049	24
5269	24
5355	24
79	25
91	25
179	25
321	25
399	25
421	25
541	25
649	25
869	25
1119	25
1229	25
1449	25
1779	25
1999	25
2219	25
2329	25
2549	25
2849	25
2975	25
3069	25
3289	25
3295	25
3399	25
3575	25
3619	25
3865	25
3919	25
4055	25
4139	25
4389	25
4499	25
4719	25
5049	25
5269	25
5355	25
Table: Files
FID	NAME	RO
2	host-defs.h	1
3	host-incs.h	1
4	c17-inline.c	0
5	prj2.c	0
Table: Filemetrics
FID	PRECPP	NCHAR	NCCOMMENT	NSPACE	NLCOMMENT	NBCOMMENT	NLINE	MAXLINELEN	MAXSTMTLEN	MAXSTMTNEST	MAXBRACENEST	MAXBRACKNEST	BRACENEST	BRACKNEST	NULINE	NPPDIRECTIVE	NPPCOND	NPPFMACRO	NPPOMACRO	NTOKEN	NSTMT	NOP	NUOP	NNCONST	NCLIT	NSTRING	NPPCONCATOP	NPPSTRINGOP	NIF	NELSE	NSWITCH	NCASE	NDEFAULT	NBREAK	NFOR	NWHILE	NDO	NCONTINUE	NGOTO	NRETURN	NASM	NTYPEOF	NPID	NFID	NMID	NID	NUPID	NUFID	NUMID	NUID	NLABEL	NMACROEXPANDTOKEN	NCOPIES	NINCFILE	NPFUNCTION	NFFUNCTION	NPVAR	NFVAR	NAGGREGATE	NAMEMBER	NENUM	NEMEMBER
2	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	16	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	18	2	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	NULL	NULL	0	1	0	0	0	0	0	0
2	1	548	367	29	0	3	22	61	47	NULL	1	1	0	0	0	5	0	0	5	37	2	0	0	2	0	3	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	2	5	8	1	1	5	7	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
3	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	15	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	0	1	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	NULL	NULL	0	0	0	0	0	0	0	0
3	1	243	187	13	0	2	13	48	28	NULL	1	1	0	0	0	1	0	0	0	16	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
4	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	45	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	144	5	20	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	12	8	NULL	60	3	2	NULL	25	0	NULL	NULL	NULL	3	2	0	0	0	0	0	0
4	1	389	0	78	0	0	10	40	45	NULL	1	1	0	0	0	0	0	0	0	143	5	20	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	6	4	0	30	3	2	0	25	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
5	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	7	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	18	3	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4	0	NULL	4	4	0	NULL	4	0	NULL	NULL	NULL	1	0	2	0	0	0	0	0
5	1	95	32	17	2	0	11	20	13	NULL	1	1	0	0	0	1	0	0	1	21	3	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4	0	1	5	4	0	1	5	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
Table: FileProj
FID	PID
2	24
3	24
4	24
1	25
2	25
3	25
4	25
5	25
Table: LineProj
FID	PID	LNUM
2	24	1
2	24	2
2	24	3
2	24	4
2	24	5
2	24	6
2	24	7
2	24	8
2	24	9
2	24	10
2	24	11
2	24	12
2	24	13
2	24	14
2	24	15
2	24	16
2	24	17
2	24	18
2	24	19
2	24	20
2	24	21
2	24	22
3	24	1
3	24	2
3	24	3
3	24	4
3	24	5
3	24	6
3	24	7
3	24	8
3	24	9
3	24	10
3	24	11
3	24	12
3	24	13
4	24	1
4	24	2
4	24	3
4	24	4
4	24	5
4	24	6
4	24	7
4	24	8
4	24	9
4	24	10
2	25	1
2	25	2
2	25	3
2	25	4
2	25	5
2	25	6
2	25	7
2	25	8
2	25	9
2	25	10
2	25	11
2	25	12
2	25	13
2	25	14
2	25	15
2	25	16
2	25	17
2	25	18
2	25	19
2	25	20
2	25	21
2	25	22
3	25	1
3	25	2
3	25	3
3	25	4
3	25	5
3	25	6
3	25	7
3	25	8
3	25	9
3	25	10
3	25	11
3	25	12
3	25	13
4	25	1
4	25	2
4	25	3
4	25	4
4	25	5
4	25	6
4	25	7
4	25	8
4	25	9
4	25	10
5	25	1
5	25	2
5	25	3
5	25	4
5	25	5
5	25	6
5	25	7
5	25	8
5	25	9
5	25	10
5	25	11
Table: LinePos
FID	FOFFSET	LNUM
2	0	1
2	4	2
2	51	3
2	54	4
2	100	5
2	119	6
2	122	7
2	184	8
2	231	9
2	234	10
2	284	11
2	288	12
2	289	13
2	321	14
2	349	15
2	378	16
2	397	17
2	416	18
2	417	19
2	467	20
2	479	21
2	523	22
3	0	1
3	4	2
3	51	3
3	54	4
3	78	5
3	97	6
3	100	7
3	149	8
3	153	9
3	154	10
3	155	11
3	196	12
3	240	13
4	0	1
4	47	2
4	97	3
4	152	4
4	210	5
4	262	6
4	317	7
4	369	8
4	424	9
4	479	10
5	0	1
5	20	2
5	34	3
5	49	4
5	58	5
5	59	6
5	80	7
5	81	8
5	89	9
5	107	10
5	109	11
Table: Definers
PID	CUID	BASEFILEID	DEFINERID
25	5	5	2
25	5	5	4
Table: Includers
PID	CUID	BASEFILEID	INCLUDERID
24	2	2	1
24	4	3	1
24	4	4	1
25	2	2	1
25	2	2	1
25	4	3	1
25	4	4	1
25	5	3	1
25	5	5	1
Table: Providers
PID	CUID	PROVIDERID
24	2	2
25	2	2
25	2	2
25	5	5
Table: IncTriggers
PID	CUID	BASEFILEID	DEFINERID	FOFFSET	LEN
25	5	5	2	471	4
25	5	5	4	7	3
Table: Functions
ID	NAME	ISMACRO	DEFINED	DECLARED	FILESCOPED	FID	FOFFSET	FANIN
821	foo	0	1	1	0	5	81	0
1669	foo2	0	1	1	1	4	166	0
2739	foo3	0	1	1	0	4	273	0
3809	foo4	0	1	1	0	4	380	0
4715	main	0	0	1	0	2	471	0
4939	foo5	0	1	1	1	4	493	0
5355	_cscout_dummy1	0	1	1	1	2	535	1
Table: FunctionDefs
FUNCTIONID	FIDBEGIN	FOFFSETBEGIN	FIDEND	FOFFSETEND
821	4	94	4	96
1669	4	207	4	209
2739	4	314	4	316
3809	4	421	4	423
4939	4	534	4	536
5355	2	557	2	577
Table: FunctionMetrics
FUNCTIONID	PRECPP	NCHAR	NCCOMMENT	NSPACE	NLCOMMENT	NBCOMMENT	NLINE	MAXLINELEN	MAXSTMTLEN	MAXSTMTNEST	MAXBRACENEST	MAXBRACKNEST	BRACENEST	BRACKNEST	NULINE	NPPDIRECTIVE	NPPCOND	NPPFMACRO	NPPOMACRO	NTOKEN	NSTMT	NOP	NUOP	NNCONST	NCLIT	NSTRING	NPPCONCATOP	NPPSTRINGOP	NIF	NELSE	NSWITCH	NCASE	NDEFAULT	NBREAK	NFOR	NWHILE	NDO	NCONTINUE	NGOTO	NRETURN	NASM	NTYPEOF	NPID	NFID	NMID	NID	NUPID	NUFID	NUMID	NUID	NLABEL	NMACROEXPANDTOKEN	NGNSOC	NMPARAM	NFPARAM	NEPARAM	FANIN	FANOUT	CCYCL1	CCYCL2	CCYCL3	CSTRUC	CHAL	IFLOW
821	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	0	0	0	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	1	0	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	18	NULL	0	NULL	0	0	1	1	1	0.0	0.0	0.0
821	1	3	0	2	0	0	1	2	0	NULL	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	NULL	0	0	0	1	1	1	0.0	0.0	0.0
1669	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	0	0	0	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	1	0	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	12	NULL	2	NULL	0	0	1	1	1	0.0	0.0	0.0
1669	1	3	0	2	0	0	1	2	0	NULL	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	NULL	0	0	0	1	1	1	0.0	0.0	0.0
2739	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	0	0	0	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	1	0	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	14	NULL	2	NULL	0	0	1	1	1	0.0	0.0	0.0
2739	1	3	0	2	0	0	1	2	0	NULL	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	NULL	0	0	0	1	1	1	0.0	0.0	0.0
3809	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	0	0	0	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	1	0	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	16	NULL	2	NULL	0	0	1	1	1	0.0	0.0	0.0
3809	1	3	0	2	0	0	1	2	0	NULL	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	NULL	0	0	0	1	1	1	0.0	0.0	0.0
4939	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	0	0	0	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	1	0	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	17	NULL	2	NULL	0	0	1	1	1	0.0	0.0	0.0
4939	1	3	0	2	0	0	1	2	0	NULL	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	NULL	0	0	0	1	1	1	0.0	0.0	0.0
5355	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	5	0	0	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	5	1	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	NULL	1	0	1	NULL	1	0	NULL	11	NULL	0	NULL	1	1	1	1	1	1.0	0.0	1.0
5355	1	22	0	3	0	0	1	21	5	NULL	0	1	0	0	0	0	0	0	0	5	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	1	0	1	0	1	0	0	NULL	0	NULL	0	1	1	1	1	1	1.0	0.0	1.0
Table: FunctionId
FUNCTIONID	ORDINAL	EID
821	0	79
1669	0	1119
2739	0	2219
3809	0	3289
4715	0	421
4939	0	4389
5355	0	5355
Table: Fcalls
SOURCEID	DESTID
5355	5355
Done
